controller/src/main/java/org/jboss/as/controller/transform/description/TransformationDescription.java

```diff
@@ -174,5 +189,27 @@ public static TransformersSubRegistration register(TransformationDescription des
             subRegistration.discardOperations(description.getDiscardedOperations().toArray(new String[description.getDiscardedOperations().size()]));
             return subRegistration;
         }
+
+        /**
+         * Register a transformation description as a sub-resource at a given {@linkplain SubsystemRegistration}.
+         *
+         * @param description the subsystem transformation description
+         * @param registration the subsystem registrations
+         * @param range the model version range the transformation applies to
+         * @return the create sub registration
+         */
+        public static TransformersSubRegistration register(TransformationDescription description, SubsystemTransformerRegistration registration, ModelVersionRange range) {
+            final TransformersSubRegistration subRegistration = registration.registerModelTransformers(range, description.getResourceTransformer(),
```

Comment:
Perhaps some wrapper around registration to avoid code duplication with the other register() method?